client/rpcserver/rpcserver_test.go

```diff
@@ -225,7 +225,7 @@ func newLink() *tLink {
 var tPort = 5555
 
 func newTServer(t *testing.T, start bool, user, pass string) (*RPCServer,
-	*TCore, func()) {
+	*TCore, func(), error) {
```

Comment:
This function uses `t.Fatalf` if any errors are encountered. `err` will always be nil for the caller.

Reply:
Thanks for pointing out the difference between fatal & error.